src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -699,4 +699,7 @@ The default is 'true' if a runtime identifier is specified.</value>
   <data name="SelfContainAndNoSelfContainedConflict" xml:space="preserve">
     <value>The '--self-contained' and '--no-self-contained' options cannot be used together.</value>
   </data>
+  <data name="DisableBuildServersOptionDescription" xml:space="preserve">
+    <value>Disable the use of build servers.</value>
```

Comment:
```suggestion
    <value>Force the command to ignore any persistent build servers.</value>
```

what do you think of this phrasing @sfoslund? I'm patterning it mostly off of the phrasing for `CommandInteractiveOptionDescription` on line 671, aiming for consistent tone overall.

Reply:
Makes sense to me, just updated to that string.